README.md

```diff
@@ -300,17 +300,27 @@ entire tree equally:
   ```
   This is useful when there's no "external parameter".
 
-- asynchronously at the end of the frame:
+- asynchronously at the end of the frame. Alternative 1:
   ```dart
   initState() {
     super.initState();
     Future.microtask(() =>
-      Provider.of<Foo>(context).fetchSomething(someValue);
+      Provider.of<Foo>(context, listen: false).fetchSomething(someValue);
     );
   }
   ```
   It is slightly less ideal, but allows passing parameters to the mutation.
```

Comment:
Instead of another `-`, what about a "Alternatively, instead of `Future.microtask`, you can use [addPostFrameCallback](https://api.flutter.dev/flutter/scheduler/SchedulerBinding/addPostFrameCallback.html)"


I'm not sure that a complete example is necessary. To begin with, this is a workaround and should be avoided as much as possible.

So I'm not too fan of this approach taking more space on the readme

Reply:
Ok, I will close this PR, but why `addPostFrameCallBack` should be avoided and what difference between `Future.microtask`?

Reply:
It's not that `addPostFrameCallback` should be avoided, but the entire idea of "delay changes at the end of the frame to bypass the initState limitation"